src/layout.rs

```diff
@@ -17,11 +17,23 @@ pub struct Layout {
 }
 
 impl Layout {
-    /// Creates a new [`Layout`] struct with zero size positioned at the origin
+    /// Creates a new zero-[`Layout`].
+    ///
+    /// The Zero-layout has size and location set to ZERO.
+    /// Order is set to 0.
     #[must_use]
     pub const fn new() -> Self {
         Self { order: 0, size: Size::ZERO, location: Point::ZERO }
     }
+
+    /// Creates a new zero-[`Layout`] from order.
+    ///
+    /// Nodes with a higher order should be rendered on top of those with a lower order.
+    /// The Zero-layout has size and location set to ZERO.
+    #[must_use]
+    pub const fn from_order(order: u32) -> Self {
```

Comment:
```suggestion
    pub const fn with_order(order: u32) -> Self {
```

Reply:
From is typically reserved for constructors that meaningfully consume their input. In this case, it's just an integer that parameterizes the constructor.

Reply:
Didn't know that, but that makes sense :)